packages/combobox/combobox.scss

```diff
@@ -151,7 +153,12 @@ $_combobox-search-input-selection-color--inverted: color.scale(
         outline: none;
         align-self: end;
         flex: 1 1 1ch;
-        height: 25px;
+        max-height: 28px;
+        padding-right: var(--jkl-combobox-input-padding);
+    }
+
+    &__search-input.menu-closed {
```

Comment:
Vi ønsker å holde oss til `BEM`-notasjon for klassenavn i Jøkul. At menyen er åpen eller lukket høres ut som en modifier på `.jkl-combobox` for meg. Så typ:

```scss
.jkl-combobox {
  // Annen CSS her

  &--menu-closed {
    .jkl-combobox__search-input {
      display: none;
    }
  }
}
```

og

```jsx
<InputGroup
  className={cn("jkl-combobox", className, {
      "jkl-combobox--invalid": !!errorLabel || invalid,
      "jkl-combobox--menu-closed": !showMenu && selectedValue.length >= 1,
  })}
```